Week2/HW2/3.initializingStatement.js

```diff
@@ -0,0 +1,8 @@
+// Adding a statement, initializing, and explaining it through console.log 
+
+let x;
+console.log("the value of my variable x will be a number");
```

Comment:
Ah, do you think it's a number `now` ? I know it `will` be a number, but what is the value right now?